components/contribution-flow/index.js

```diff
@@ -160,6 +160,7 @@ class ContributionFlow extends React.Component {
       isSubmitting: false,
       showSignIn: false,
       createdOrder: null,
+      forceSummaryStep: this.getCurrentStepName() !== STEPS.DETAILS, // If not starting the flow with the details step, we force the summary step to make sure contributors have an easy way to review their contribution
```

Comment:
This is how we're resolving https://github.com/opencollective/opencollective/issues/6313